modules/platform/rest-api/nuxeo-rest-api-test/src/test/java/org/nuxeo/ecm/restapi/test/AuditTest.java

```diff
@@ -420,6 +421,64 @@ public void shouldEnrichWithLatestDocumentLogEntries() throws IOException {
         }
     }
 
+    @Test
+    public void shouldHandleSortingAndPagination() throws Exception {
+        DocumentModel doc = RestServerInit.getFile(1, session);
+
+        List<LogEntry> logEntries = new ArrayList<>();
+        logEntries.add(buildLogEntry(doc, "Two", "secondEvent", "james", null));
+        logEntries.add(buildLogEntry(doc, "Two", "firstEvent", "james", null));
+        logEntries.add(buildLogEntry(doc, "Two", "thirdEvent", "james", null));
+        logEntries.add(buildLogEntry(doc, "Two", "thirdEvent", "james", null));
+        logEntries.add(buildLogEntry(doc, "Two", "thirdEvent", "james", null));
+        logEntries.add(buildLogEntry(doc, "Two", "thirdEvent", "james", null));
+        logEntries.add(buildLogEntry(doc, "One", "secondEvent", "james", null));
+        logEntries.add(buildLogEntry(doc, "One", "thirdEvent", "james", null));
+        logEntries.add(buildLogEntry(doc, "One", "firstEvent", "james", null));
+        logEntries.add(buildLogEntry(doc, "One", "thirdEvent", "james", null));
+        logEntries.add(buildLogEntry(doc, "One", "firstEvent", "james", null));
```

Comment:
Sounds weird to have "firstEvent" or "thirdEvent" occur several times.
Maybe call them "eventA", "eventB", ...

Reply:
Yes, It was my first idea, but I tried to be consistent with the exiting tests where the pattern is to have this kind of name. But I can change them